System-Testing-Deployments-and-Upgrades/User-Authorization/index.rst

```diff
@@ -0,0 +1,162 @@
+
+.. |author| replace:: *Tiago Ribeiro*
+.. |contributors| replace:: *Wouter van Reeven*, *Rob Bovill*
+
+.. _User-Authorization:
+
+##################
+User Authorization
+##################
+
+The Rubin Observatory Control System has an authorization system designed to help manage and coordinate user interaction with the system.
+
+By default, the system is designed to be operating under autonomous CSC control.
+For example, during survey operations the Scheduler will compute the observing queue and feed them to the ScriptQueue, which then executes the SAL Scripts required to observe the target.
+
+At the same time, users may want to interact with the system to perform engineering or other tasks that require a finer level of interaction with the system, which is usually done from :ref:`nublado <Control-System-User-Interfaces-Nublado>`.
```

Comment:
consider making this two sentences:
At the same time, users may want to interact with the system to perform engineering tasks that require a finer level of interaction. Such activities often utilize :ref:`Nublado <Control-System-User-Interfaces-Nublado>`.